$OpenBSD: patch-src_utils_cpp,v 1.3 2014/10/03 10:24:42 dcoppa Exp $

Use the Adwaita icon theme unconditionally: fixes a problem with
minitube GUI not having icons when executed outside of a Desktop
Environment

--- src/utils.cpp.orig	Mon Sep 23 11:30:27 2013
+++ src/utils.cpp	Wed Sep 25 13:22:21 2013
@@ -24,6 +24,7 @@ $END_LICENSE */
 #endif
 
 QIcon Utils::themeIcon(const QString &name) {
+    QIcon::setThemeName("Adwaita");
     if (QIcon::hasThemeIcon(name))
         return QIcon::fromTheme(name);
     else
